pkg/console/controllers/clidownloads/controller.go

```diff
@@ -188,6 +188,41 @@ The oc binary offers the same capabilities as the kubectl binary, but it is furt
 	}
 }
 
+func LicenseDownloads(host, cliDownloadsName string) *v1.ConsoleCLIDownload {
```

Comment:
Also, this function is never called by the controller, so the license download is not being created.

Reply:
You will need to add something like this to call it:

```golang

// up by the oc & odo calls
ocLicenseCLIDownloads := LicenseDownloads(host, api.OCCLIDownloadsCustomResourceName+"license")
_, ocLicenseCLIDownloadsReason, ocLicenseCLIDownloadsErr := ApplyCLIDownloads(c.consoleCliDownloadsClient, ocLicenseCLIDownloads)
status.HandleDegraded(updatedOperatorConfig, "OCDownloadsSync", ocLicenseCLIDownloadsReason, ocLicenseCLIDownloadsErr)
if ocLicenseCLIDownloadsErr != nil {
	return ocLicenseCLIDownloadsErr
}
```